```python
from pprint import pprint

import {{{packageName}}}
from {{{packageName}}}.rest import ApiException

configuration = stencila.hub.Configuration(
    host="https://hub.stenci.la/api",
    api_key={'Token': 'YOUR_API_TOKEN'},
    api_key_prefix={'Token': 'Token'}
)
with {{{packageName}}}.ApiClient(configuration) as api_client:
    api_instance = {{{packageName}}}.{{{classname}}}(api_client)
    {{#allParams}}
    {{paramName}} = {{{example}}} # {{{dataType}}} | {{{description}}}{{^required}} (optional){{/required}}{{#defaultValue}} (default to {{{.}}}){{/defaultValue}}
    {{/allParams}}
    try:
        {{#returnType}}api_response = {{/returnType}}api_instance.{{{operationId}}}({{#allParams}}{{#required}}{{paramName}}{{/required}}{{^required}}{{paramName}}={{paramName}}{{/required}}{{#hasMore}}, {{/hasMore}}{{/allParams}}){{#returnType}}
        pprint(api_response){{/returnType}}
    except ApiException as e:
        print("Exception when calling {{classname}}.{{operationId}}: %s\n" % e)
```
